homeassistant/components/kodi/media_player.py

```diff
@@ -311,6 +306,9 @@ def __init__(self, connection, kodi, name, uid):
         self._media_position = None
         self._connect_error = False
 
+        self._attr_name = name
+        self._attr_supported_features = SUPPORT_KODI
```

Comment:
```suggestion
```
This was changed in a later PR and is not needed anymore (noticed it after rebase)

Reply:
thanks for catching that!